### ISSUE TITLE
Electron - Clusterize not defined

### ISSUE THREAD

**Issue by xmrhahnx** [NeXTs/Clusterize.js#178]
Hi there,
i'm trying to use Clusterize.js in my Electron app. 
If I run my app with node, it works like a charm but when I switch to Electron I obtained the error `Uncaught (in promise) ReferenceError: Clusterize is not defined` in the console.

Any suggestion?

**Comment by xmrhahnx** [comment]
Solved!
Some refactoring of the Clusterize.js code solves the issue!

**Comment by 8483** [comment]
@xmrhahnx   

How did you manage to get it working in node?  

I am doing this, but it won't work.

```bash
npm i clusterize.js
```

```js
var Clusterize = require("Clusterize");


var clusterize= new Clusterize({
  rows: data,
  scrollId: "scrollArea",
  contentId: "contentArea"
});
```

**Comment by xmrhahnx** [comment]
@8483 I used it on the js client side, so I just included the .js and the .css file like shown in "Include it on your page" section of the guide [https://clusterize.js.org/](https://clusterize.js.org/). Have you  some errors in console?

**Comment by liujinliang01** [comment]
Hi xmrhahnx,
I am trying to replicate the demo at this page: https://clusterize.js.org/, using the local<src> option, but I get an error message at the DevTools console saying that Clusterize is not defined.
Could you please share your refactored clusterize.js?
Thank you in advance.
best regards
Jim

**Comment by liujinliang01** [comment]
> Hi xmrhahnx, I am trying to replicate the demo at this page: https://clusterize.js.org/, using the local option, but I get an error message at the DevTools console saying that Clusterize is not defined. Could you please share your refactored clusterize.js? Thank you in advance. best regards Jim

Update on my own previous Question:
My problem is solved mysteriously - I tried to use the clusterize.js and then the min file. Neither worked.  Searched for a solution for a few days but could not find any. But then, today, I copied the clusterize.js file from https://github.com/NeXTs/Clusterize.js/. This time it worked! When I compared the version I downloaded before and the version I just copied, I find no difference between the two. I don't know why the same version did not work before. Mystery.

Jim
